client/src/locales/en-US.json

```diff
@@ -278,7 +278,8 @@
             "keyringFailedTitle": "Login failed",
             "keyringFailedQuestion": "Could not login using the system authentication. Would you like to use a recovery file?",
             "keyringFailedUsedRecovery": "Use a recovery file",
-            "keyringFailedAbort": "No"
+            "keyringFailedAbort": "No",
+            "vaultFailed": "Could not log in using Parsec Account."
```

Comment:
A little too generic IMO, is it possible to add a little more detail about what could have happened?